### PR TITLE
Add Bandsintown and Nextcloud Cookbook & rework K9-Mail

### PR DESCRIPTION
**Pull request by peterge1998**
## Description

add bandsintown

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)

### Icons added
com.bandsintown
com.fsck.k9
com.nextcloud_cookbook_flutter

### CODE REVIEW

**Comment by peterge1998** [comment]
#766

**Comment by peterge1998** [comment]
K9-Mails Icon is fine? It's the second Icon I designed...
image

**Review by Chefski — CHANGES_REQUESTED** [review]
Hey! Could you add the SVGs for the icons to the SVG folder?

**Comment by peterge1998** [comment]
> Hey! Could you add the SVGs for the icons to the SVG folder?

Done :)

**Comment by peterge1998** [comment]
This should be way better. Feel free to criticize :)
Thank you very much for your very helpful tips, here & in the issue!

**Review by Chefski — CHANGES_REQUESTED** [review]
One last tweak needed. The rectangle at the bottom has points that are no aligned horiaztontally. Could you align them and then it should be fine :)

image

**Comment by peterge1998** [comment]
Oh, that's because I started that shape in the top left corner and just closed it when i set the top right point. Can I do this with Inkscape too?

**Comment by Chefski** [comment]
Use a guideline to align it on the same X axis

**Comment by peterge1998** [comment]
Okay, I still need to generate the XML with android studio, but I was able to do it with the Node tool :)

**Comment by peterge1998** [comment]
So, now its ready to merge @Chefski :)

File: app/src/main/res/xml/grayscale_icon_map.xml
Comment (Chefski) [inline app/src/main/res/xml/grayscale_icon_map.xml:129]: Should this line be added back but linked to the Etar icon or is it an invalid package name?

**Review by Chefski — CHANGES_REQUESTED** [review]
New icons look fine. Could you just change `etar.svg` to have a content area size of 144x144px as its a square. After that I'll merge it and you can open new PRs when you add icons.

image

**Comment by peterge1998** [comment]
> New icons look fine. Could you just change `etar.svg` to have a content area size of 144x144px as its a square. After that I'll merge it and you can open new PRs when you add icons.
> 
> image

Okay, if i did the math correctly it should now be fine!

**Review by Chefski — APPROVED** [review]
Thanks! :D

**Comment by daywalk3r666** [comment]
https://github.com/LawnchairLauncher/lawnchair/commit/aa251677483fde500a0f34fd31e761bdb44566c7

**Comment by Chefski** [comment]
@daywalk3r666 thanks for bringing that up.

**Pull request merged**
